### PR TITLE
Bump docker to version 10.2.2

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: alpine/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/alpine/alpine.tmpl.dockerfile.
 FROM postgres:13.4-alpine
```
Comment (hanefi) [inline alpine/Dockerfile:3]: I think it is time we bump the alpine base image to PG 14

```suggestion
FROM postgres:14.0-alpine
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+### citus-docker v10.2.2.docker (October 14,2021) ###
+
+* Bump Citus version to 10.2.2
```
Comment (hanefi) [inline CHANGELOG.md:3]: Maybe we should add an entry for alpine base image bump.

```suggestion
* Bump Citus version to 10.2.2

* Bump alpine image PG version to 14.0
```

Comment (gurkanindibay) [reply CHANGELOG.md:3]: Fixed as you suggested

File: Dockerfile [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
 FROM postgres:14
```
[the remark this answers is not in the document]
Comment (gurkanindibay) [reply Dockerfile:3]: I think we should keep major version to be able to get benefit from base images minor update

**Pull request merged**